qiskit/pulse/transforms/alignments.py

```diff
@@ -44,6 +46,22 @@ def align(self, schedule: Schedule) -> Schedule:
         """
         pass
 
+    @abc.abstractmethod
+    def set_sequence(self, sequence: PyDAG, **kwargs) -> None:
```

Comment:
I am not sure kwargs is the way to go here. The difficulty is that parallel alignment needs the mixed frames mapping, but sequential alignment doesn't. It's also easier to just send the mapping to all alignment types, without checking the alignment type.